packages/react/src/ErrorBoundary.en.mdx

```diff
@@ -48,27 +48,33 @@ const ErrorAfter4s = () => {
 
 ![Example banner](/gif/errorboundary-example.gif)
 
-#### Importing External Components
+:::tip
+
+## Define component as `<ErrorBoundary/>`'s fallback
+
+### ErrorBoundaryFallbackProps
 
-If you want to deliver a declared component to fallback, you can use the `ErrorBoundaryFallbackProps` type to declare the component.
+If you want to deliver a declared component as `<ErrorBoundary/>`'s fallback, you can use the `ErrorBoundaryFallbackProps` type to declare the component easily.
 
 ```tsx
-import { ErrorBoundaryFallbackProps } from '@suspensive/react'
+import type { ErrorBoundaryFallbackProps } from '@suspensive/react'
```

Comment:
I followed our eslint rule to express ErrorBoundaryFallbackProps as type definitely